Makefile

```diff
@@ -19,34 +19,20 @@ conda-freeze:  ## Use conda to freeze the current env available
 conda-install-frozen:  ## Use conda to install dev dependencies using pinned env specification - subject to repodata.json changes
 	conda env create -f environment-frozen.yml --force --name jupyterlab-code-formatter
 
-dev-install-serverextension:  ## Use pip to install the server extension in dev mode
-	pip install -e serverextension
-	jupyter serverextension enable --py jupyterlab_code_formatter
-
-dev-install-labextension:  ## Use npm to install the lab extension in dev mode
-	cd $(LABEXTENSION_PATH)
-	npm install
-	npm run build
-	jupyter labextension link .
-	cd -
-
-dev-install: dev-install-serverextension dev-install-labextension  ## Install both lab and server extension in dev mode
+dev-install:
+	pip install -e .
+	jupyter labextension develop . --overwrite
+	jlpm run build
```

Comment:
I consolidated the dev install into one and removed `dev-watch-jupyterlab` to reflect the current recommendations from the cookiecutter: https://github.com/jupyterlab/extension-cookiecutter-ts/tree/master/%7B%7Bcookiecutter.python_name%7D%7D#development-install